code/ai/ai_profiles.cpp

```diff
@@ -472,6 +472,8 @@ void parse_ai_profiles_tbl(const char *filename)
 
 				set_flag(profile, "$use actual primary range:", AI::Profile_Flags::Use_actual_primary_range);
 
+				set_flag(profile, "$fix Guard Attack This bug:", AI::Profile_Flags::Fix_guard_attack_this_bug);
```

Comment:
Since this is a sensible fix this should be enabled by default for newer mods using the supported version feature. Maybe you could add a version of `set_flag` that accepts a default value that you then initialize with the return value of `mod_supports_version()`.

Reply:
Good idea, looks like there is a nice section that Goober made specifically for bug fixes, so I'll add it to that.